src/subsplit_dag.hpp

```diff
@@ -495,6 +495,43 @@ class SubsplitDAG {
     size_t cur_node_count;
     // Current edge count after modification.
     size_t cur_edge_count;
+
+    void Reinit(const SubsplitDAG &dag) {
+      prv_node_count = dag.NodeCount();
+      prv_edge_count = dag.EdgeCountWithLeafSubsplits();
+      node_reindexer = Reindexer::IdentityReindexer(prv_node_count);
+      edge_reindexer = Reindexer::IdentityReindexer(prv_edge_count);
+      added_node_ids = NodeIdVector();
+      added_edge_idxs = EdgeIdVector();
+    }
+
+    ModificationResult ComposeWith(const ModificationResult other) {
```

Comment:
Not a huge deal, but is there a reason these code blocks are sitting in the hpp file?

Reply:
No, I can put that in the cpp file.